akka-actor-typed-tests/src/test/scala/docs/akka/typed/StyleGuideDocExamples.scala

```diff
@@ -371,18 +371,23 @@ object StyleGuideDocExamples {
     //#public-private-messages-2
     // above example is preferred, but this is possible and not wrong
     object Counter {
-      sealed trait PrivateCommand
-      sealed trait Command extends PrivateCommand
+      // The type of all public and private messages the Counter actor handles
+      sealed trait Message
+
+      /** Counter's public message protocol type. */
+      sealed trait Command extends Message
       case object Increment extends Command
       final case class GetValue(replyTo: ActorRef[Value]) extends Command
       final case class Value(n: Int)
 
-      // Tick is a PrivateCommand so can't be sent to an ActorRef[Command]
-      case object Tick extends PrivateCommand
+      // The type of the Counter actor's internal event messages.
+      private sealed trait Event extends Message
+      // Tick is a private Event so can't be sent to an ActorRef[Command]
+      private case object Tick extends Event
```

Comment:
since this example is about the alternative approach to `private` the text doesn't correspond to the example any more. I think it would be better to not use private visibility in this example, since that is covered by the first example.